samples/k8s-connected-and-managed-cluster/README.md

```diff
@@ -0,0 +1,155 @@
+
+Sample below indicates all distributions of connectedClusters type and all ditributions of managedClusters type are supported.
+
+![Alt text](images/Manifest_changes.PNG)
+
+Following is the guidance on the supportedClusterTypes:
+
+| Property      | Description |
+| -----------   | ----------- |
+| supportedClusterTypes       | Contains an object for each top-level cluster-type. Allowed types are – “managedClusters”, “connectedClusters”. "managedClusters" denotes Azure Kubernetes Service (AKS) clusters. "connectedClusters" denotes Azure Arc-enabled Kubernetes clusters. For each of these cluster types, specify distributions and unsupported Kubernetes versions for these distributions. If supportedClusterTypes is not provided, all distributions of ‘managedClusters’ will be supported by default. If supportedClusterTypes is provided, and a given top level cluster type is not provided, then all distributions and Kubernetes versions for that cluster type will be treated as unsupported. |
+| distribution   | An array of distributions  corresponding to the cluster type. Provide name(s) of specific distributions. Set the value to [“All”]  to indicate all distributions are supported. |
+| distributionSupported  | A boolean value representing whether the specified distribution(s) are supported. If false, providing a value for UnsupportedVersions will cause an error. |
+| unsupportedVersions  | A list of versions for the specified distribution(s) which are unsupported. Supported operators: |
+
+                        •	"=" Given version is not supported. E.g.: “=1.2.12”
+
+                        •	">" All versions greater than the given version are not supported. E.g.: “>1.1.13”
+
+                        •	"<" All versions less than the given version are not supported. E.g.: “<1.3.14”
+
+                        •	"..." All versions in range are unsupported. E.g.: ” 1.1.2...1.1.15”  (includes right-side value and excludes left-side value)
+
+#### More samples of supportedClusterTypes:
+Only the "AKS" distribution of connectedClusters type is supported with restriction on some versions. All distributions of managedClusters is supported with no restriction on versions.
+
+``` yaml
+  supportedClusterTypes:
+    connectedClusters:
+      - distribution: ["AKS"]
+        distributionSupported: true
+        unsupportedVersions: ["=1.2.0"]
+    managedClusters:
+      - distribution: ["All"]
+        distributionSupported: true
+        unsupportedVersions: null
+```
+
+Only the "AKS" distribution of connectedClusters type is supported with restriction on some versions. Only the "AKS" distribution of managedClusters is supported with restriction on some versions.
+
+``` yaml
+  supportedClusterTypes:
+    connectedClusters:
+      - distribution: ["AKS"]
+        distributionSupported: true
+        unsupportedVersions: ["1.10.0...1.20.0", ">1.21.1"]
+    managedClusters:
+      - distribution: ["AKS"]
```

Comment:
What is really the difference in AKS vs All in managedClusters?

Reply:
Both are same at this time as we don't have any other flavor of managedClusters.